sdk/test/trace/tracer_provider_test.cc

```diff
@@ -25,9 +25,14 @@ TEST(TracerProvider, GetTracer)
   ASSERT_EQ(t1, t3);
 
   // Should be an sdk::trace::Tracer with the processor attached.
-  auto sdkTracer = dynamic_cast<Tracer *>(t1.get());
-  ASSERT_NE(nullptr, sdkTracer);
-  ASSERT_EQ(processor, sdkTracer->GetProcessor());
+  auto sdkTracer1 = dynamic_cast<Tracer *>(t1.get());
```

Comment:
Because the `Tracer` interface in `api` does't have functions `GetProcessor()` and `Getsampler()`. I need to cast it to sdk tracer to check its processor & sampler.